### PR TITLE
[ALLUXIO-2739] Fix "runMiniBentchmark" command to respect master addr set in Configuration 

### PR DESCRIPTION
**Pull request by jswudi**
https://alluxio.atlassian.net/browse/ALLUXIO-2739

### CODE REVIEW

**Comment by jswudi** [comment]
PTAL

File: examples/src/main/java/alluxio/cli/MiniBenchmark.java [hunk 1]
```diff
@@ -64,18 +64,17 @@ private static void usage() {
...
   public static void main(String[] args) throws Exception {
-    if (args.length != 4) {
+    if (args.length != 3) {
       usage();
       System.exit(-1);
     }
-    AlluxioURI masterLocation = new AlluxioURI(args[0]);
-    Configuration.set(PropertyKey.MASTER_HOSTNAME, masterLocation.getHost());
-    Configuration.set(PropertyKey.MASTER_RPC_PORT, masterLocation.getPort());
+    Configuration.set(PropertyKey.MASTER_HOSTNAME, Configuration.get(PropertyKey.MASTER_HOSTNAME));
```
Comment (apc999) [inline examples/src/main/java/alluxio/cli/MiniBenchmark.java]: 71 and 72 are no-op---you get a value and set it back. It is fine to remove these two lines

File: examples/src/main/java/alluxio/cli/MiniBenchmark.java [hunk 2]
```diff
@@ -64,18 +64,17 @@ private static void usage() {
...
+    Configuration.set(PropertyKey.MASTER_RPC_PORT, Configuration.get(PropertyKey.MASTER_RPC_PORT));
     FileSystemContext.INSTANCE.reset();
 
-    OperationType operationType = Enum.valueOf(OperationType.class, args[1]);
-    long fileSize = FormatUtils.parseSpaceSize(args[2]);
-    int iterations = Integer.parseInt(args[3]);
+    OperationType operationType = Enum.valueOf(OperationType.class, args[0]);
```
Comment (apc999) [inline examples/src/main/java/alluxio/cli/MiniBenchmark.java]: you can simplify this to be:
```java
OperationType operationType = OperationType.valueOf(args[0]);
```

**Comment by apc999** [comment]
Thanks for the fix. I left two minor comments. 
Overall it looks good to me.

**Pull request merged**